app/geo/gui/src/ts/twxplore/gui/geo/states/map/MapFilterVariables.ts

```diff
@@ -0,0 +1,11 @@
+export interface MapFilterVariables {
+  frequency: {
+    min: number | null;
```

Comment:
I usually don't use null unless I need an explicit placeholder indicating that something is absent. Having undefined (min?: number) makes life easier.